yoyodyne/schedulers.py

```diff
@@ -88,63 +130,28 @@ def add_argparse_args(parser: argparse.ArgumentParser) -> None:
         "--start_factor",
         type=float,
         default=1 / 3,
-        help="Start factor for lr decay. Default: %(default)s.",
+        help="The starting multiplier for the lr in LinearDecay. Default: %(default)s.",
```

Comment:
suggest "LR"; also on lines 138 and 145.

Above, for  warmup steps, it reads "(warmupinvsqrt scheduler only)". Here it reads "in LinearDecay". I don't care which style we use (yours may be better actually) but we should standardize.